src/index.js

```diff
@@ -30,6 +30,10 @@ class TravelTime extends LitElement {
     this.googleMapsBaseUrl = 'https://maps.google.com/?';
     this.googleSearchByLL = 'daddr=';
     this.googleSearchByQuery = 'daddr=';
+
+    this.hereBaseUrl = 'https://wego.here.com/directions/mix//';
```

Comment:
two slashes at the end here

Reply:
That is needed so the coordinates/name get interpreted as the destination.
The origin is after the first slash and empty if not given